### PR TITLE
add pascals-triangle exercise

### PR DESCRIPTION
**Pull request by anuragsoni**

### CODE REVIEW

File: exercises/pascals-triangle/pascals-triangle.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default class Triangle {
+    rows: number[][]
+    lastRow: number[]
+
+    constructor(_rows: number) {
```
[the remark this answers is not in the document]
Comment (masters3d) [reply exercises/pascals-triangle/pascals-triangle.ts]: Lets remove the constructor and variables. Lets just leave the main signature on

File: exercises/pascals-triangle/pascals-triangle.example.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    lastRow: number[]
+
+    constructor(rows: number) {
+        this.rows = this.fillRows(rows)
+        this.lastRow = this.rows[this.rows.length - 1]
+    }
+
+    newRow(prevRow: number[]) {
+        let prev = 0
+        const nextRow = []
```
[the remark this answers is not in the document]
Comment (masters3d) [reply exercises/pascals-triangle/pascals-triangle.example.ts]: can you address this?

[the remark this answers is not in the document]
Comment (anuragsoni) [reply exercises/pascals-triangle/pascals-triangle.example.ts]: this has already been addressed? its const now

File: exercises/pascals-triangle/pascals-triangle.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (anuragsoni) [reply exercises/pascals-triangle/pascals-triangle.ts]: Hmm I have addressed the linter comment by ensuring the block isn't empty, rows is empty.

If I remove the constructor and variables, that would mean that when someone tries to run npm-test the first time they won't see a test failure, they will just see that function signature isn't correct as the test tries to use the constructor. Is that okay?

**Comment by anuragsoni** [comment]
@masters3d I have pushed an update addressing your comment about the constructor

**Comment by masters3d** [comment]
Thanks!

**Pull request merged**